migrations/message_store_postgres/content_script_version_6.nim

```diff
@@ -1,24 +1,22 @@
 const ContentScriptVersion_6* =
   """
 -- Rename old table
-ALTER TABLE IF EXISTS MESSAGES
+ALTER TABLE MESSAGES
 RENAME TO OLD_MESSAGES;
 
 -- Remove old message index
-ALTER TABLE IF EXISTS OLD_MESSAGES
+ALTER TABLE OLD_MESSAGES
 DROP CONSTRAINT MESSAGEINDEX;
 
 -- Create new empty table
-CREATE TABLE IF NOT EXISTS NEW_MESSAGES (
+CREATE TABLE MESSAGES (
 	MESSAGEHASH VARCHAR NOT NULL,
 	PUBSUBTOPIC VARCHAR NOT NULL,
 	CONTENTTOPIC VARCHAR NOT NULL,
 	PAYLOAD VARCHAR,
 	VERSION INTEGER NOT NULL,
 	TIMESTAMP BIGINT NOT NULL,
 	META VARCHAR,
-	ID VARCHAR,
-	STOREDAT BIGINT,
 	CONSTRAINT MESSAGEINDEX PRIMARY KEY (TIMESTAMP, MESSAGEHASH)
```

Comment:
@SionoiS - should we establish the PK as only the `MESSAGEHASH`?

Reply:
Would it make sense to create an unique index per message hash to speed up store queries using message hash crtiteria?

Reply:
The initial comment that I did is outdated because we will follow a different migration approach